lib/matplotlib/tests/test_widgets.py

```diff
@@ -59,6 +59,46 @@ def test_rectangle_selector():
     check_rectangle(props=dict(fill=True))
 
 
+@pytest.mark.parametrize('spancoords', ['data', 'pixels'])
+@pytest.mark.parametrize('minspanx, x1', [[0, 10], [1, 10.5]])
+@pytest.mark.parametrize('minspany, y1', [[0, 10], [1, 10.5]])
```

Comment:
```suggestion
@pytest.mark.parametrize('minspanx, x1', [[0, 10], [1, 10.5], [1, 11], [1, 11.5]])
@pytest.mark.parametrize('minspany, y1', [[0, 10], [1, 10.5], [1, 11], [1, 11.5]])
```

Is it worth adding more cases:
- when it is equal to the `mispan` values
- when it is larger and then selector shouldn't be removed

Reply:
Good shout with the first one. For the second suggestion I've left it out because:
- In would complicate the test
- `click_and_drag(tool, start=(20, 20), end=(30, 30))` already checks that a rectangle with `0 < minspan < span` is drawn correctly

Reply:
> * `click_and_drag(tool, start=(20, 20), end=(30, 30))` already checks that a rectangle with `0 < minspan < span` is drawn correctly

Yes, indeed!